plugins/extractors/superset/typeDef.go

```diff
@@ -1,18 +1,48 @@
 package superset
 
 type Dashboard struct {
-	ID          int `json:"id"`
```

Comment:
@scortier We already dashboard model in proton. We should use that.

Reply:
Yes will use that format only , can we add more model top over the prescribed format ?

Reply:
Do you mean more fields?

Reply:
Yes

Reply:
@ravisuhag this model is just to catch superset model when fetching from its API.

This Dashboard model will then be mapped to our meteor model before sending to our stream.

Reply:
or maybe we can rename it to `SupersetModel` to avoid confusion.